packages/grid/_modules_/grid/components/pagination.tsx

```diff
@@ -11,6 +12,16 @@ export interface PaginationComponentProps {
   rowsPerPageOptions?: number[];
 }
 
+// Used to hide the drop down select from the TablePaginagion
+const useStyles = makeStyles({
```

Comment:
Ok, I've updated the logic with what we discussed during the planning today, here is the summary:
 - Remove `Total Rows` when the pagination is available
 - Remove the select and the label inside the `TablePagination`
 - Update the login so the `selected rows` is only shown on desktop when the pagination is available 
 - Adjust the footer items alignment when there is only pagination (it should be aways on the right side)

Last but not least a question - these styles are needed to hide the select from the `TablePagination` component but I'm not sure it this is the best place to keep them, ideas ?